tests/field/example-django.sh

```diff
@@ -40,6 +48,7 @@ done
 
 log 2 'Create demo project from scratch and push it'
 tox -e cookiecutter -- \
+    -o /tmp/painless-generated-project \
```

Comment:
I see that `tox` creates subfolders in that folder for the projects. Hence, I'd suggest use the plural (`-projects`).